README.md

```diff
@@ -6,7 +6,7 @@
 1. What is the DOM?
 2. What is an event?
 3. What is an event listener?
-4. What is a Constructor
+4. What is a Constructor?
```

Comment:
You should make an actual pull request to Ivan for this! Let's talk about how to do this with the team on Monday. :)